vpr/src/base/read_options.cpp

```diff
@@ -2009,7 +2013,7 @@ argparse::ArgumentParser create_arg_parser(std::string prog_name, t_options& arg
             " * criticality: Sets the minimum budgets to 0 and the maximum budgets as a function of delay and criticality (net delay/ pin criticality) [EXPERIMENTAL]\n"
```

Comment:
This help on options doesn't seem to match the option values below. Can you update it?

Reply:
Sure thing, a simple next PR could be to give RCV it's own distinct option in the router, but for now I'll update this.